sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -415,6 +415,12 @@ _previous AS (
     *
   FROM
     `moz-fx-data-shared-prod.fenix_derived.firefox_android_clients_v1`
+  WHERE
+    {% if is_init() %}
+      FALSE
+    {% else %}
+      submission_date < @submission_date
```

Comment:
@fbertsch I believe we should be doing this filter inside `_previous`?

Reply:
This filter inside `_previous` returns data until the current date to avoid data with future dates - which happens when the ping sends the timestamps based on the client's date, and this one has been modified by the user to be in the future.
For initialization there is no need to compare with previous data, therefore using WHERE FALSE.
Does that resolve tour question @kik-kik?

Reply:
Hey @lucia-vargas-a , in the current version of the query that is not there. This is what I'm adding in this PR which would do exactly how you described in your comment. The main question is, was this something that was omitted previously and should be added or should not be there to begin with?

Reply:
It was just omitted initially, which shouldn't affect the results.
I think it's a good addition to limit the data queried and reduce costs. Also, if there is interest in identifying data with dates in the future, this can be done in the ping's table directly.